templates/dashboards/grafana-dashboard-image-builder-worker-general.configmap.yml

```diff
@@ -2487,8 +2605,8 @@ data:
             "auto_min": "10s",
             "current": {
               "selected": false,
-              "text": "6h",
-              "value": "6h"
+              "text": "7d",
+              "value": "7d"
```

Comment:
keep this at 6h by default, this determines the granularity/the amount of the data points shown; 6h is a happy medium between smoothing out and showing spikes.